homeassistant/components/environment_canada/sensor.py

```diff
@@ -74,6 +71,8 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
 class ECSensor(Entity):
     """Implementation of an Environment Canada sensor."""
 
+    SCAN_INTERVAL = timedelta(minutes=10)
```

Comment:
It should be on module level. Now it's on class level. So up top of file after imports.

Reply:
Got it, thanks.